redaxo/src/core/backend.php

```diff
@@ -195,6 +195,17 @@
 // include the requested backend page
 rex_be_controller::includeCurrentPage();
 
+// update body class if minibar has been set inactive
+rex_extension::register('OUTPUT_FILTER', function (rex_extension_point $ep) {
+    if (rex_minibar::getInstance()->isActive() === false) {
+        $ep->setSubject(preg_replace(
+            '/(<body.*|<html.*)rex-minibar-is-active(.*>)/i',
```

Comment:
Wenn ich das richtig sehe, matcht der Regex auch auf Folgendes:

```html
<body>
    <p>Bitte verwende "rex-minibar-is-active".</p>
</body>

Reply:
Danke für den Hinweis, @gharlan. Ich habe es nun darauf angepasst, dass nur zwischen `<body` oder `<html` und dem ersten schließenden `>` gematcht wird. Nicht perfekt, dafür simpel und in dem Kontext okay, finde ich.